### ISSUE TITLE
Deployment fails because /usr/bin/env: node: No such file or directory

### ISSUE THREAD

**Issue by ngw** [platanus/capistrano-bower#7]
Hi, I'm deploying a Rails app that makes use of bower and nvm. The problem I'm having now, is that it appears that Capistrano is not running nvm.sh, without being able to find node during bower:install.
Does anyone know what the problem is here?

Capistrano output:
https://gist.github.com/ngw/5e6491071ef1ee364cebcd61b1347d38

Gemfile.lock:
https://gist.github.com/ngw/8b5b862915c288f618cd3420e707b598

deploy.rb:
https://gist.github.com/ngw/1010135afbd02824a37987be1a1d360f

Any idea what's happening here? Is it using the capistrano-nvm gem? Doesn't appear so...

**Comment by blackjid** [comment]
You can try adding this to your `deploy.rb`

``` ruby
set :nvm_map_bins, %w{node npm bower}
```

**Comment by ngw** [comment]
Eh, it was already there, and I removed it. It fails exactly as before, like capistrano-bower doesn't care about it.
Just to be clear, something that works is moving up the nvm.sh inclusion inside bashrc to happen when the shell is non-interactive, which unfortunately breaks the deployment of errbit (and is basically wrong).
This is why I started suspecting capistrano-bower, even thought the code appears to be perfect.

**Comment by blackjid** [comment]
mm, I don't know... We use http://github.com/platanus/capistrano-nodenv instead of nvm... and it work, but I don't think thats the problem, because the implementation are the same, and both taken from http://github.com/capistrano/rbenv

How your `Capfile` looks like?

**Comment by ngw** [comment]
Here is my Capfile:
https://gist.github.com/ngw/a659531f20398588ccf2865f99d7edd5

I also don't know what I can eventually do to debug this, everything else works with capistrano-nvm, but it appears that for some reasons capistrano-bower doesn't work with it

**Comment by blackjid** [comment]
Yes, but I don't see other node stuff running, just bower. Why do you say that everything else works??

This is how bower is being called

```
INFO [e40784fe] Running /usr/bin/env /usr/local/bin/bower install --quiet --config.interactive=false on xxx.xxx.xxx.xxx
```

Look at the path `/usr/local/bin/bower` if `capistrano-nvm` was working, bower should be called from the nvm path and the specified node version.

I think the problem is with `capistrano-nvm` not this gem... This gem expects that bower is in the path to execute it, that can be achieved by using capistrano-nvm, capistrano-nodev, or as you said tweaking the system environment.

**Comment by hosamshahin** [comment]
This solved the problem for me.

ln -s /usr/bin/nodejs /usr/bin/node
ln -s /usr/bin/nodejs /usr/sbin/node
